tests/jballerina-unit-test/src/test/resources/test-src/bala/test_bala/functions/test_global_function_pointers.bal

```diff
@@ -1,4 +1,4 @@
-import testorg/foo;
+import testorg/foo as foo;
```

Comment:
We don't need this, right?
```suggestion
import testorg/foo;
```

Reply:
Yes. But it's clear(for me :smiley:).

Reply:
Oh okay, :smiley:  But if you think of how we get the prefix (part of the module name after the last period) and given that  the module name has no periods here, there's no difference, right?

This is the same as saying
```suggestion
import ballerina/io as io;
```

Reply:
Removed.:relaxed: